toqito/state_props/tests/test_is_abs_ppt.py

```diff
@@ -0,0 +1,71 @@
+"""Unit tests for the is_abs_ppt function in toqito.state_props."""
+
+import numpy as np
+import pytest
+
+from toqito.rand import random_psd_operator
+from toqito.state_props import is_abs_ppt
+from toqito.states import max_mixed
+
+
+def test_maximally_mixed_states():
+    """Test that maximally mixed states are absolutely PPT for supported dimensions."""
+    # 2x2
+    rho_2x2 = max_mixed(4)
+    assert is_abs_ppt(rho_2x2, [2, 2]) is True
+    # 2x3
+    rho_2x3 = max_mixed(6)
+    assert is_abs_ppt(rho_2x3, [2, 3]) is True
+    # 3x3
+    rho_3x3 = max_mixed(9)
+    assert is_abs_ppt(rho_3x3, [3, 3]) is True
```

Comment:
if a unit test is following the same pattern, it is better to use https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions